package.json

```diff
@@ -43,7 +43,7 @@
     "throttleit": "^1.0.0",
     "unlimited": "^1.1.0",
     "webtorrent": "0.x",
-    "webtorrent-app": "feross/webtorrent-app",
+    "webtorrent-desktop": "git+ssh://git@github.com/feross/webtorrent-desktop.git",
```

Comment:
Might be able to just use the github releases API. That would give us access to latest version info and download assets.